week2/04_log_and_register_custom_model.py

```
@@ -0,0 +1,136 @@
+import json
+from mlflow import MlflowClient
+from mlflow.utils.environment import _mlflow_conda_env
+import subprocess
+
+def adjust_predictions(predictions, scale_factor=1.3):
+    return predictions * scale_factor
+
+def get_git_sha():
+    try:
+        return subprocess.check_output(['git', 'rev-parse', 'HEAD']).decode('ascii').strip()[:7]
+    except:
```

Ruff (0.7.0)

Do not use bare `except`

(E722)